### ISSUE TITLE
Overly verbose LTI course instance linking page

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#10901]
When viewing the [LTI 1.3 course instance linking page](https://github.com/PrairieLearn/PrairieLearn/blob/ee5e6b22a761e05f81a1eeb603a5a8382793d89f/apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts) page, instructors with a lot of courses have the actionable items (the select and save button) pushed off the bottom of the screen, with no indication that the user should scroll down. The list of courses doesn't seem to play any role here and I think it should be deleted.

As well as the main issue above, this page includes a lot of information before the actionable items. My observation from watching instructors use this page is that it would be easier for them if we moved the select+save above all the info text. I also think we could reduce the info text to something like "If you need a PrairieLearn course then [request one here]. If you need a new course instance, create one in your existing course in [PrairieLearn]".

### Current page screenshot

![Image](https://github.com/user-attachments/assets/1efcd067-9acf-434e-b88f-74df299baa50)

**Comment by mfsilva22** [comment]
yes, please move any action item to the top of the page. We are often frustrated because students don't read instructions? I think we are all guilty of that :-) The list of courses is not useful and just clutter the page.
